worker_specific_task_queues/README.md

```diff
@@ -0,0 +1,56 @@
+# Worker-Specific Task Queues
+
+Use a unique Task Queue for each Worker in order to have certain Activities run on a specific Worker. In the Go and Java SDKs, this is explicitly supported via the Session option, but in other SDKs a different approach is required.
```

Comment:
There are no sessions in Java. I should have caught this when I reviewed the user's PR.